include/graphic/frame.h

```diff
@@ -0,0 +1,22 @@
+#ifndef GRAPHIC_FRAME_H
+#define GRAPHIC_FRAME_H
+
+#include <vector>
+#include <graphic/color.h>
+
+class Frame {
+private:
+    int verticalResolution;
+    int horizontalResolution;
+    std::vector<std::vector<Color>> matrix;
```

Comment:
Creio que aqui vai ser um vetor de RGBs não normalizados, certo? Se for esse o caso, usa o `Pixel` aqui, quando o PR #24 for para main, tu atualiza essa branch e usa o `Pixel` p já subir tudo nos conformes

Reply:
beleza

Reply:
Bora pra jogo essa alteração @kinhosz

Reply:
Feito